src/components/SignOutButton/SignOutButton.jsx

```diff
@@ -0,0 +1,49 @@
+import {useEasyauth} from '../../hooks/useEasyauth/useEasyauth.jsx';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export const SignOutButton = ({callback, children}) => {
+  const auth = useEasyauth();
+  if (!children) {
+    return (
+      <a
+        href={
+          process.env.REACT_APP_EASYAUTH_APP_URL +
```

Comment:
take these from the initial configuration of EasyauthProvider somehow. Make useEasyauth access these somehow?

Reply:
resolved in commit 2f7699f2d46f4a589f1b2438dfaf2866a8dd7c72.